src/IW/Effects/Download.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE AllowAmbiguousTypes  #-}
+
+module IW.Effects.Download where
```

Comment:
It's better to write the list of exports in each module. Here the list is not going to be huge anyway :slightly_smiling_face: